torch/onnx/symbolic_opset9.py

```diff
@@ -521,6 +522,25 @@ def reciprocal(g: jit_utils.GraphContext, self):
 @symbolic_helper.parse_args("v", "i")
 @_beartype.beartype
 def cat(g: jit_utils.GraphContext, tensor_list, dim):
+    """
```

Comment:
Those functions do not manipulate tensors, they convert the corresponding pytorch function into onnx operators. I think it should be made explicit if you plan to document  these function. They are not meant to be called directly by a user.

Reply:
Thank you. I adjusted the docstrings according to your comment.